english/eating-walls/marshmallows-for-breakfast/eat-breakfast.md

```diff
@@ -0,0 +1 @@
+Do you want to get breakfast[get breakfast](../breakfast/breakfast.md)
```

Comment:
You are right. This link [get breakfast] (../breakfast/breakfast.md) was my first attempt to link to my story. But when Casey said it's broken, I replaced it with [get breakfast](../english/breakfast/breakfast.md) which fixed the broken link problem.
I tested the new link which lives in Udacity's marshmallow.md file and I was able to reach my story.
I hope it's ok for merge now.

Thanks for you real-time collaboration.  


     On Tuesday, June 9, 2015 6:33 AM, Caroline Buckey <notifications@github.com> wrote:
   

 In english/eating-walls/marshmallows-for-breakfast/eat-breakfast.md:> @@ -0,0 +1 @@
> +Do you want to get breakfast[get breakfast](../breakfast/breakfast.md)
It looks to me like this was a previous attempt to add a link to breakfast.md, before you realized that you needed to add the link to an existing file, such as marshallow.md. Is that accurate? If so, and you no longer want this page, you can remove it using the command git rm.—
Reply to this email directly or view it on GitHub.